internal/retry/retry.go

```diff
@@ -401,6 +401,7 @@ func (ebo ExpBackOff) nextWait(attempts int) time.Duration {
 	if ebo.Jitter {
 		// Choose a random value in [0, 2^attempt) with uniform probability.
 		// The expected value is (1/2)*2^attempt = 2^(attempt-1).
+		//nolint: gosec
 		s := time.Duration(rand.Int63n(1 << attempts))
```

Comment:
What does linter not like here?

Reply:
```
internal/retry/retry.go:405:22: G404: Use of weak random number generator (math/rand instead of crypto/rand) (gosec)
                s := time.Duration(rand.Int63n(1 << attempts))
```